aspnetcore/security/authentication/scaffold-identity/consoleAddUser/Data/SeedData.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading.Tasks;
+using WebApplication1.Data;
+
+// dotnet aspnet-codegenerator razorpage -m Contact -dc ApplicationDbContext -outDir Pages\Contacts --referenceScriptLibraries
+namespace ContactManager.Data
+{
+    #region snippet
+    public static class SeedData
+    {
+        public static async Task Initialize(IServiceProvider serviceProvider, string testUserPw)
+        {
+            using (var context = new ApplicationDbContext(
+                serviceProvider.GetRequiredService<DbContextOptions<ApplicationDbContext>>()))
+            {
+                // Password is set with the following:
+                // dotnet user-secrets set SeedUserPW <pw>
```

Comment:
I would avoid providing a specific implementation for this and simply mention that this should be done carefully without going into much detail, but it likely involves creating a random password and a distributing the password to the user through some other mechanism.